src/js/common/components/form-elements/ErrorableCheckbox.jsx

```diff
@@ -90,7 +90,7 @@ ErrorableCheckbox.propTypes = {
   checked: React.PropTypes.bool,
   errorMessage: React.PropTypes.string,
   name: React.PropTypes.string,
-  label: React.PropTypes.string.isRequired,
+  label: React.PropTypes.object.isRequired,
```

Comment:
Yes, though only tangentially related to this PR. We sometimes pass rendered components as checkbox labels (so we can use links) and hadn't ever changed this prop type.